inst/examples/dashboard/app.R

```diff
@@ -18,7 +19,11 @@ examplePages <- imap(examples, function(example, name) {
 })
 names(examplePages) <- NULL
 
-pages <- c(list(route("/", homePage)), examplePages)
+pages <- c(list(route("/", homePage)),
+  list(route("about", aboutPage)),
+  examplePages
+)
```

Comment:
Slightly cleaner:
```suggestion
pages <- c(
  list(
    route("/", homePage),
    route("about", aboutPage)
  ),
  examplePages
)
```

Reply:
Thanks I didn't know I could do this. FIXED!!